src/elements/common/flowTypes.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+type ErrorType = {
+    code: string,
+    details?: Object,
```

Comment:
Should this have a flow type as well to denote the shape of the details object?

Reply:
There isn't a specific shape